src/wordpress-playground/mu-plugins/my-hacks.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Various hacks to the WordPress Playground experience.
+ */
+
+// Add a notice to wp-login.php offering the username and password.
+add_action(
+	'login_message',
+	function() {
+		return <<<EOT
+<div class="message info">
+	<strong>username:</strong> <code>admin</code><br /><strong>password</strong>: <code>password</code>
```

Comment:
It's fine to hardcode this for now. In the future it would be great to configure that at WordPress build time and then display the same credentials here. Not a blocker by any means – let's track this in a separate issue.

Reply:
Created https://github.com/WordPress/wordpress-playground/issues/138